crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -120,6 +122,7 @@ impl SuiTransactionBlockResponseOptions {
             show_events: true,
             show_object_changes: true,
             show_balance_changes: true,
+            show_raw_effects: true,
```

Comment:
this will break existing clients that use `full_content`.

Reply:
What do you recommend then? To have `show_raw_effects: false` here? Then it's not longer full content..

Reply:
yeah this wasn't written in a way that is future proof unfortunately. But from (existing) users' perspective, if they don't expect this field, it's fine to mark it as false. We can add a special comment explaining it

Reply:
Addressed in the latest commit.